test/app-route.html

```diff
@@ -59,19 +87,32 @@
 <script>
   'use strict';
 
+  var cooldownFunction = function() {};
+  if (/^Apple/.test(navigator.vendor)) {
+    cooldownFunction = function(done) {
```

Comment:
this had to be added because of added shadowdom tests. See the original test/iron-location